src/kademlia/dht_storage.cpp

```diff
@@ -505,6 +505,20 @@ namespace
 			return m_counters;
 		}
 
+		dht_storage_items save_items(int const /*max_items*/) const override
+		{
+			dht_storage_items ret;
+
+			// TODO: implement here
+
+			return ret;
+		}
+
+		void load_items(dht_storage_items const& /*data*/) override
```

Comment:
I would imagine these items could be moved in here. the caller of this function won't pass these items to anyone else, right?